man/systemd.network.xml

```diff
@@ -212,6 +212,12 @@
           below 1280 (the minimum MTU for IPv6) it will automatically be increased to this value.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>ARP=</varname></term>
+        <listitem>
+          <para> A boolean. Configures ARP protocol for a given interface. Defaults to unset.</para>
```

Comment:
Maybe it should be clarified that "unset" means "no change from kernel defaults", rather than "ARP is disabled by default"? The current wording is a little scary.